bsp/nrf/radio.c

```diff
@@ -30,7 +30,7 @@
 #define RADIO_INTERRUPT_PRIORITY 1
 #endif
 
-#define RADIO_TIFS          150U  ///< Inter frame spacing in us
+#define RADIO_TIFS          0U  ///< Inter frame spacing in us. zero maens IFS is enforced by software, not the hardware
```

Comment:
```suggestion
#define RADIO_TIFS          0U  ///< Inter frame spacing in us. zero means IFS is enforced by software, not the hardware
```

The datasheet also states:
"When (Fast RU is) enabled, TIFS is not enforced by hardware and software needs to control when to turn on the Radio."
How/when do you control when to turn on the radio?

Reply:
I believe this means that if you have TIFS enabled, the hardware will physically delay the TX_READY event when you try to send packets too quickly, in order to respect some specific interframe spacing.
I believe this was meant to respect some 802.15.4 specification. We are not using that specification, so I don't think this affects our code.